Resources/Prototypes/Entities/Objects/Weapons/Guns/Projectiles/projectiles.yml

```diff
@@ -1114,6 +1114,6 @@
     impactEffect: BulletImpactEffectRedDisabler
     damage:
       types:
-        Heat: 20 # Slightly more damage than the 17heat from the Captain's Hitscan lasgun
+        Heat: 20
```

Comment:
pro you undid the buff

Reply:
as I said in response to lyndo I don't want to see energy guns be wield only and have a fire rate reduction, which is what direction asked for. I don't necessarily agree with their reasoning but direction has spoken, the new intent is to just raise mag cap and make ik60 auto fire.